container/webui/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: MIT
 #!BuildTag: openqa_webui:latest opensuse/openqa-webui:latest opensuse/openqa-webui:%PKG_VERSION% opensuse/openqa-webui:%PKG_VERSION%.%RELEASE%
-FROM opensuse/leap:15.5
+FROM opensuse/leap:15.6
```

Comment:
See https://github.com/os-autoinst/openQA/actions/runs/9008630845/job/24768938320#step:3:4155

According to `zypper info --requires apache2` apache2-utils (and hence `gensslcert`) is no longer a required package for apache2 in Leap 15.6.

I suggest a small change on line 21 in this Dockerfile

```diff
- zypper in -y ca-certificates-mozilla curl openQA-local-db apache2 hostname which w3m 'rubygem(sass)' && \
+ zypper in -y ca-certificates-mozilla curl openQA-local-db apache2 apache2-utils hostname which w3m 'rubygem(sass)' && \
```

Reply:
sure, done